pretty-simple.cabal

```diff
@@ -36,13 +36,14 @@ library
                      , Text.Pretty.Simple.Internal.Color
                      , Text.Pretty.Simple.Internal.Expr
                      , Text.Pretty.Simple.Internal.ExprParser
-                     , Text.Pretty.Simple.Internal.ExprToOutput
-                     , Text.Pretty.Simple.Internal.Output
                      , Text.Pretty.Simple.Internal.OutputPrinter
   build-depends:       base >= 4.8 && < 5
                      , ansi-terminal >= 0.6
                      , containers
                      , mtl >= 2.2
+                     , prettyprinter >= 1.6.1
+                     , prettyprinter-ansi-terminal >= 1.1.1.2
+                     , Stream
```

Comment:
We almost certainly don't want this dependency - it just made one of the functions easier to implement as a proof-of-concept.

Reply:
[It](http://hackage.haskell.org/package/Stream) doesn't look too problematic to me actually. The transitive dependency on `lazysmallcheck` isn't ideal, but very tiny and apparently quite stable too.

Reply:
Removed, as explained in a comment below